src/qt/combobox.cpp

```diff
@@ -190,6 +190,18 @@ bool wxComboBox::IsReadOnly() const
     return HasFlag( wxCB_READONLY );
 }
 
+bool wxComboBox::IsEditable() const
+{
+    return !IsReadOnly() && !m_qtComboBox->lineEdit()->isReadOnly();
```

Comment:
It's a bit suspicious that we check for it being null below, but not here. Shouldn't we?